pkg/upgraders/upgradeable.go

```diff
@@ -0,0 +1,65 @@
+package upgraders
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/go-logr/logr"
+	configv1 "github.com/openshift/api/config/v1"
+	upgradev1alpha1 "github.com/openshift/managed-upgrade-operator/pkg/apis/upgrade/v1alpha1"
+)
+
+func (c *clusterUpgrader) IsUpgradeableVersion(ctx context.Context, logger logr.Logger) (bool, error) {
+	v := &configv1.ClusterVersion{}
+	currentVersion, err := c.cvClient.GetClusterVersion()
```

Comment:
You can use cvClient's `GetCurrentVersion()` here instead, which I'd recommend. You can use this to pass directly into the `semver.Parse()` call below.

Reply:
When I was looking at `GetCurrentVersion()` function, it takes as input the value from `GetClusterVersion()` thus I kept both. https://github.com/openshift/managed-upgrade-operator/blob/master/pkg/clusterversion/cv.go#L205. Please let me know, if it's correct in the latest commit.

Reply:
That looks great, thank you!